control/controlexecute/progress.go

```diff
@@ -39,11 +39,11 @@ func (p *ControlProgressRenderer) Start() {
 	defer p.updateLock.Unlock()
 
 	if p.enabled {
-		p.spinner = display.ShowSpinner("Starting controls")
+		p.spinner = display.ShowSpinner("Starting controls...")
```

Comment:
we show starting controls twice - can the first one be changed to `Initializing...`